### PR TITLE
Upgrade to netcoreapp3.1 and standardize logging

### PR DESCRIPTION
**Pull request by sprucely**
There was a drawn-out game of musical chairs in getting npm packages configured and playing nicely. Well mostly nicely, as it appears hmr is not working. If anything looks like an odd choice, I can revisit it.

### CODE REVIEW

**Comment by sprucely** [comment]
Started working on azure pipeline and then realized I hadn't run the karma tests. Gotta get those passing again...

**Review by srkirkland — APPROVED** [review]

**Comment by srkirkland** [comment]
I think it all looked good from a code perspective.  Once the JS tests are updated we'll need to do our own testing before we deploy but everything looked like a good approach to me.

**Review by jSylvestre — CHANGES_REQUESTED** [review]
Failing locally after CAS login.
Needs to be tested once this is fixed

File: Anlab.Mvc/Client/components/Summary.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export default class Summary extends React.PureComponent<ISummaryProps, {}> {
     private _renderSubmitButton = () => {
         if (this.props.placingOrder) {
             return (
-                <Button className="btn btn-order" disabled={!this.props.canSubmit || this.props.isSubmitting} onClick={this.props.onSubmit}>
+                <Button className="btn btn-order" disabled={!this.props.canSubmit || this.props.isSubmitting} onClick={() => this.props.onSubmit()}>
```
[the remark this answers is not in the document]
Comment (sprucely) [reply Anlab.Mvc/Client/components/Summary.tsx:95]: Just saw this question. Right, it was just to satisfy typescript.

**Pull request closed without merging**